src/Xrefcheck/Verify.hs

```diff
@@ -117,6 +115,9 @@ instance (Given ColorMode, Buildable a) => Buildable (WithReferenceLoc a) where
     #{wrlItem}
     |]
 
+newtype DomainName = DomainName {unDomainName :: Text}
+  deriving stock (Show, Eq, Ord)
```

Comment:
Good :+1: 

Please also write a brief haddock description, maybe with an example (it is the simplest way to be concrete here I think). So that there would be no ambiguities in interpreting the semantics of this type.

And, spacing [does not match](https://github.com/serokell/style/blob/master/haskell.md#general-identifiers) the style guide :sweat_smile: